### DISCUSSION TITLE
Is it possible to keep the Terminal's session?

### DISCUSSION THREAD

**Discussion by i11010520** [LazyVim/LazyVim#4072] [Q&A] [answered]
I toggled on Terminal, run some long-run job, then toggled off.
After a while, when I re-toggled on Terminal, a new Terminal session popped up, and I can't check my long-run job.:(
As `ps` shows, the job was running somewhere.
But how can I get back the the previous Terminal?

**Comment by folke** [comment, marked as answer]
`<C-/>` to toggle a terminal. Use a count for another temrinal

**Reply by i11010520** [reply]
I did toggle the terminal using `<C-/>` (actually is `<C-_>`, cause `<C-/>` has no effect on my machine).
But the previous terminal session can be restored only when toggling interval time is short.

**Comment by i11010520** [comment]
I test a few times, it works.
Suspecting whether the unrestored terminal toggle was due to some oblivious `count`.
